person.rb

```diff
@@ -21,6 +21,7 @@ def correct_name
 
   # define getter&setter for @name & @age
   attr_accessor :name, :age
+  attr_reader :rental
```

Comment:
Great job on implementing the `rental` getter method! It looks like you've made progress towards meeting the project requirements for creating the `many-to-many` (also `has-many-through`) relationship between `Person` and `Book` using the intermediate class `Rental`. Your code seems to be on the right track, but there are a couple of points to consider for further improvement and alignment with the specified requirements:


1. **Adding Rentals:**
   The requirement mentions that there should be a method for adding rentals to a person. You can create a method like `add_rental(book, date)` in your `Person` class.


Remember, you're making good progress! Feel free to refer to the provided resources for additional guidance on implementing the necessary methods and relationships. Keep up the good work!

Best regards,
Temitope
Code Reviewer
01/09/2023

---